CI/linux-qt6/sort_ui/sort_ui.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
```

Comment:
you could add this as a submodule instead of copying

Reply:
I was considering it, but thought a copy would be better since the original repo has 3 stars and the fork none. So the repos might just get deleted at some point.